pkg/horizon/deployment.go

```diff
@@ -56,9 +56,9 @@ func Deployment(instance *horizonv1.Horizon, configHash string, labels map[strin
 		},
 	}
 	readinessProbe := &corev1.Probe{
-		TimeoutSeconds:      25,
```

Comment:
I'm assuming the timeout higher than the period wasn't intentional e.g to ensure multiple requests hit the service concurrently. @bshephar do you know?